fiftyone/utils/iou.py

```
@@ -485,42 +527,98 @@
+    preds,
+    gts,
+    iscrowd=None,
+    classwise=False,
+    sparse=False,
+):
     is_symmetric = preds is gts
 
+    if sparse:
+        ious = defaultdict(list)
+    else:
+        ious = np.zeros((len(preds), len(gts)))
```

Ruff

Use ternary operator `ious = defaultdict(list) if sparse else np.zeros((len(preds), len(gts)))` instead of `if`-`else`-block

Replace `if`-`else`-block with `ious = defaultdict(list) if sparse else np.zeros((len(preds), len(gts)))`

(SIM108)